src/Shared/Recruit.Vacancies.Client/Infrastructure/TableStore/StorageRetryPolicy.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Polly;
+using Polly.Retry;
+
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.TableStore
+{
+    public static class StorageRetryPolicy
+    {
+        public static RetryPolicy GetRetryPolicy(ILogger logger)
+        {
+            return Policy
```

Comment:
Wider question... I believe we are not using the azure inbuilt retry policies for consistency purpose?

Reply:
Yes. Since Polly is being used elsewhere in the solution, i too have used polly. I got this confirmed from Mark.